grammars/jsdoc.cson

```diff
@@ -411,6 +435,120 @@
         ]
       }
     ]
+  # Default @param value, highlighted as JavaScript
+  'default-value':
+    'patterns': [
+      # Double-quoted string
+      {
+        'begin': '(")'
```

Comment:
Same thing here.  The double `string.quoted.double.js` capture/contentName feel _extremely_ hacky to me, and is actually why it's taken me so long to get back to reviewing this PR.

Is there any other way to do this without the double capture?